server/api/impl/store/apps.go

```diff
@@ -16,6 +16,7 @@ func (s *Store) ListApps() []*apps.App {
 	}
 	for _, v := range conf.Apps {
```

Comment:
Why use `ListApps` for all apps and `LoadApp` for a single one? 1/5 to use `GetListedApps` and `GetListedApp` instead. Also, `StoreApp` is a bit redundant (since called on a store), 1/5 to use `SaveApp` instead (or as I was pointing down below, get rid off all App/Manifest suffixes and have more than one store wrapper).